tests/travis/install-db2-ibm_db2.sh

```diff
@@ -4,6 +4,9 @@ set -ex
 
 echo "Installing extension"
 (
+    # https://travis-ci.community/t/then-sudo-apt-get-update-failed-public-key-is-not-available-no-pubkey-6b05f25d762e3157-in-ubuntu-xenial/1728/12
+    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6B05F25D762E3157
```

Comment:
```
$ sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6B05F25D762E3157
Executing: /tmp/tmp.mBcQpgQ9mB/gpg.1.sh --keyserver
keyserver.ubuntu.com
--recv-keys
6B05F25D762E3157
gpg: requesting key 762E3157 from hkp server keyserver.ubuntu.com
gpg: key DC282033: "https://packagecloud.io/github/git-lfs (https://packagecloud.io/docs#gpg_signing) <support@packagecloud.io>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1
```

How is the packagecloud.io key relevant to DB2?

Reply:
@morozov 

apt update was failing - see:

[https://travis-ci.org/doctrine/dbal/jobs/569134569](https://travis-ci.org/doctrine/dbal/jobs/569134569)

Seems to be fixed now, will close this.